### ISSUE TITLE
Implement brilliant fan expansion called Starsmith: Expanded Oracles 

### ISSUE THREAD

**Issue by themaster567** [ben/foundry-ironsworn#756]
https://www.drivethrurpg.com/product/417619/Starsmith-Expanded-Oracles

It's an amazing product that you can set any price including free for, so I think it may be safe to include. Maybe it could be an optional toggle if people don't want it by default.

Take a look at his other expansions for  both systems too. They all look like they're worthy tbh. It seems like all of them are as a "pay-what-you-want" sort of deal.

**Comment by rsek** [comment]
yeah, i hear eric does good work, and you're not the first to ask about it. you may want to talk to `aledroid` on the ironsworn discord, as iirc they've been doing some work on setting up Starsmith's tables for their own use. maybe they can hook you up?

but, IMO, the most practical way to do this would be a separate module. there's several reasons for that:

* `foundry-ironsworn` already has 300+ tables in it, for crying out loud :wink: that data adds up, and many hosting setups can put some constraints on storage. anything of this magnitude *really* needs to be opt-in so that we're not consuming a bunch of space for content that may not even be desired, and that opt-in has to happen *before* the package is downloaded. in other words, it needs to be a separate package.
* it'd 3rd party content. typically, FVTT system packages include "core"/"canonical" content, while expansions and supplements (3rd party or otherwise) are the purview of modules. i'm uneasy with creating a situation that could misrepresent its authorship. all unofficial system package development involves a certain amount of editorialization, but you gotta draw a line somewhere
  * real talk: once certain package management improvements from v11 arrive, i'm going to give serious thought to splitting out *Ironsworn: Delve* into its own module... and Delve actually *is* canonical!
* modules can be system agnostic. so people playing with other systems but wanting to use Starsmith tables would be SOL if it were integrated into this system. one of FVTT's greatest strengths is modules, and philosophically i think it'd be better to lean into that
* supporting a big 3rd party expansion is a not-insignificant scope increase, in large part because it'd probably mean maintaining another dataforged-comptatible dataset myself :grimacing: with sundered isles on the horizon and no shortage of additional features/improvements that i'd *like* to work on, it seems unlikely that i'll have any bandwidth devote to this in the foreseeable future

anyways, that's just my two cents. @ben may differ on this -- thoughts?

**Comment by themaster567** [comment]
This is all extremely well-written and goes far beyond the hopes I even had. I expected to just be rejected, so I had begun the slow process of doing everything myself by hand. If there's more people besides me working on this in any form, I am elated!
